### PR TITLE
Fix pymedusa/SickRage/issues/551

### PR DESCRIPTION
**Pull request by fernandog**
Uses the same code from here:
https://github.com/pymedusa/SickRage/blob/develop/sickbeard/server/web/home/handler.py#L947-L959

If works, we can move this code to a `helper`

@ratoaq2 @p0psicles @medariox Thanks for the inputs

### CODE REVIEW

**Comment by ratoaq2** [comment]
That should be a static method in TVEpisode from the start. :-p

**Comment by fernandog** [comment]
Also i think we should pass a list of segments (ep ojb) and not pass one segment each time.
We already loop in in queue

File: sickbeard/tv.py [hunk 1]
```diff
@@ -186,6 +186,24 @@ def __init__(self, indexer, indexerid, lang='', quality=None, flatten_folders=No
         self._load_from_db()
 
     @property
+    def season_episodes_list(self, season):
+        """Returns the episodes numbers from season
+
```
Comment (p0psicles) [inline sickbeard/tv.py]: First want to say nice job fernandog.  Second yes I think you should return list of TVEpisode() to do it proper.  As other property methods,  also return objects.

File: sickbeard/failedProcessor.py [hunk 2]
```diff
@@ -22,6 +22,7 @@
 from sickbeard import logger
 from sickbeard import show_name_helpers
 from sickbeard import search_queue
+from sickbeard.tv.TVShow import season_episodes_list
```
Comment (fernandog) [inline sickbeard/failedProcessor.py]: @ratoaq2 why I can't do this? I only want to import season_episodes_list

**Comment by fernandog** [comment]
getting this error while manual snatching season pack:

```
2016-09-04 15:12:56 DEBUG    SNATCHQUEUE-MANUALSNATCH-271910 :: [5e9a550] Traceback (most recent call last):
  File "/home/osmc/SickRage/sickbeard/search_queue.py", line 412, in run
    self.success = search.snatchEpisode(search_result)
  File "/home/osmc/SickRage/sickbeard/search.py", line 157, in snatchEpisode
    if datetime.date.today() - curEp.airdate <= datetime.timedelta(days=7):
AttributeError: 'list' object has no attribute 'airdate'
```

**Comment by fernandog** [comment]
```
2016-09-05 11:46:43 INFO     Thread-17 :: [8944860] [Fear.the.Walking.Dead.S01.BDRip.X264-REWARD.#85 : Failed download]
2016-09-05 11:46:43 INFO     Thread-17 :: [8944860] I did encounter some unprocessable items: 
2016-09-05 11:46:43 INFO     Thread-17 :: [8944860] Successfully processed
2016-09-05 11:46:43 INFO     Thread-17 :: [8944860] Failed Download Processing succeeded: (Fear.the.Walking.Dead.S01.BDRip.X264-REWARD, /media/SAMSUNG/NZBget/inter/Fear.the.Walking.Dead.S01.BDRip.X264-REWARD.#85)
2016-09-05 11:46:43 DEBUG    Thread-17 :: [8944860] Deleted folder: /media/SAMSUNG/NZBget/inter/Fear.the.Walking.Dead.S01.BDRip.X264-REWARD.#85
2016-09-05 11:46:42 INFO     Thread-17 :: [8944860] Deleting folder: /media/SAMSUNG/NZBget/inter/Fear.the.Walking.Dead.S01.BDRip.X264-REWARD.#85
2016-09-05 11:46:42 DEBUG    Thread-17 :: [8944860] name_parser info: title: Fear the Walking Dead, season: 1, format: BluRay, type: episode, absolute_episode: [], quality: Unknown, episode: []
2016-09-05 11:46:42 DEBUG    Thread-17 :: [8944860] Parsed Fear.the.Walking.Dead.S01.BDRip into title: Fear the Walking Dead, season: 1, format: BluRay, type: episode, absolute_episode: [], quality: Unknown, episode: []
2016-09-05 11:46:42 INFO     Thread-17 :: [8944860] Using nzb_name for release name.
2016-09-05 11:46:42 INFO     Thread-17 :: [8944860] Failed download detected: (Fear.the.Walking.Dead.S01.BDRip.X264-REWARD, /media/SAMSUNG/NZBget/inter/Fear.the.Walking.Dead.S01.BDRip.X264-REWARD.#85)
2016-09-05 11:46:42 DEBUG    Thread-17 :: [8944860] Processing folder Fear.the.Walking.Dead.S01.BDRip.X264-REWARD.#85
2016-09-05 11:46:42 DEBUG    Thread-17 :: [8944860] PostProcessing VideoInRar: 
2016-09-05 11:46:42 DEBUG    Thread-17 :: [8944860] PostProcessing RarContent: 
2016-09-05 11:46:42 DEBUG    Thread-17 :: [8944860] PostProcessing VideoFiles: []
2016-09-05 11:46:42 DEBUG    Thread-17 :: [8944860] PostProcessing Files: []
2016-09-05 11:46:42 DEBUG    Thread-17 :: [8944860] PostProcessing Dirs: [u'Fear.the.Walking.Dead.S01.BDRip.X264-REWARD.#85']
```

**Comment by medariox** [comment]
Approved

**Pull request merged**